BitwardenShared/UI/Vault/Vault/VaultList/VaultListSectionTests.swift

```diff
@@ -49,7 +49,19 @@ class VaultListSectionTests: BitwardenTestCase {
 
         let subjectWithCipher = [
             VaultListSection(id: "2", items: [.fixtureGroup(group: .card, count: 2)], name: "Cards"),
-            VaultListSection(id: "5", items: [.fixture(cipherView: .fixture(type: .login))], name: "Items"),
+            VaultListSection(
+                id: "5",
+                items: [
+                    .fixture(
+                        cipherListView: .fixture(
+                            type: .login(
+                                .fixture()
+                            )
```

Comment:
🎨 I think you can simplify this with the other fixture you added:
```suggestion
                        cipherListView: .fixture(login: .fixture())
```